src/Utils/request/uploadFile.ts

```diff
@@ -31,7 +31,28 @@ const uploadFile = async (
         } catch {
           error = xhr.responseText;
         }
-        Notification.BadRequest({ errs: error.errors });
+        // Extracting messages from response Text
+        let userFriendlyMessages: string[] = [];
+        if (error?.errors && Array.isArray(error.errors)) {
+          error.errors.forEach((err: any) => {
+            if (
+              err.msg?.profile_picture &&
+              Array.isArray(err.msg.profile_picture)
+            ) {
+              userFriendlyMessages = userFriendlyMessages.concat(
+                err.msg.profile_picture,
+              );
+            }
+          });
+        }
+
+        if (userFriendlyMessages.length > 0) {
+          // Combining messages into a single string
+          const message = userFriendlyMessages.join("\n• ");
+          Notification.BadRequest({ errs: `• ${message}` });
+        } else {
+          Notification.BadRequest({ errs: error.errors });
+        }
```

Comment:
look into how https://github.com/ohcnetwork/care_fe/pull/11152/files tried to solve this! I don't see any reason for all these complications to achieve something simple!

Little cleanups to how that was done should solve the issue.